src/server/svr_mail.c

```diff
@@ -134,8 +134,8 @@ svr_mailowner_id(char *jid, job *pjob, int mailpoint, int force, char *text)
 		if (pjob != 0) {
 
 			if (pjob->ji_qs.ji_svrflags & JOB_SVFLG_SubJob) {
-				if (pjob->ji_wattr[(int)JOB_ATR_mailpnts].at_flags & ATR_VFLAG_SET) {
-					if (strchr(pjob->ji_wattr[(int)JOB_ATR_mailpnts].at_val.at_str,
+				if (is_jattr_set(pjob, JOB_ATR_mailpnts)) {
```

Comment:
Can these nested ifs be merged?

Reply:
unfortunately not, it'll change the logic.